src/modules/python/pbs_hooks/PBS_alps_inventory_check.PY

```diff
@@ -155,11 +158,12 @@ def get_apstat_nids(msg):
     # not ship the subprocess module
     apstat_out = os.popen(cmd_apstat)
 
+    pattern = re.compile("(.*?)\s+(.*?)\s+(.*?)\s+(.*?)\s+(.*)")
+

```

Comment:
This is still rather permissive.  I just did a test where it matched 5 spaces.  Also, why did you use .*?
That should match 0 or more characters and the ? will match 0 or 1 of the 0 or more characters.  At the very least, the ? seems redundant.  I still think you should use .+ to say 1 or more characters.  Also since you have 5 groups, it'd be nice if you named them (?P<name>pattern) so (?P<nid>.+)